front/app/api/custom_field_statements/useCustomFieldStatements.ts

```diff
@@ -0,0 +1,76 @@
+import { UseQueryOptions, useQueries } from '@tanstack/react-query';
+
+import { ICustomFields } from 'api/custom_fields/types';
+
+import fetcher from 'utils/cl-react-query/fetcher';
+
+import customFieldStatementKeys from './keys';
+import {
+  ICustomFieldStatementParameters,
+  IFormCustomFieldStatement,
+} from './types';
+
+const fetchCustomFieldStatements = ({
```

Comment:
```suggestion
const fetchCustomFieldStatement = ({
```

Should be singular since it fetches a single statement